plugin/web/extensions/geolocation/core/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { Icon } from "@web/sharedComponents";
+import { styled } from "@web/theme";
+
+import useHooks from "./hooks";
+
+const GeolocationWrapper: React.FC = () => {
+  const { handleFlyToCurrentLocation } = useHooks();
+
+  return (
+    <Wrapper width={44} height={44}>
+      <Icon icon="sun" size={20} onClick={handleFlyToCurrentLocation} />
+    </Wrapper>
+  );
+};
+
+export default GeolocationWrapper;
+
+const Wrapper = styled.div<{ width?: number; height?: number }>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-shrink: 0;
+  cursor: pointer;
+  overflow: hidden;
+  position: relative;
```

Comment:
Is this needed?

Reply:
right , removed

Reply:
fc8de4437577d111f90b0a1c7db09d37c48d4de6